### PR TITLE
Shayan / upgrade ChartIQ to version 8.1

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

**Review by zaki-binary — DISMISSED** [review]

File: src/Translation.js [hunk 1]
```diff
@@ -33,13 +33,23 @@ export class Translation {
     translate(...args) {
         const curr_lang = lang_map[this.lang];
         const key = args[0].trim();
-        let translated = curr_lang ? (curr_lang[key] || key) : key;
+        const key_with_qoutation = key.replace(/"/gi, '\\\"'); /* eslint-disable-line */
+        let translated = key;
+        let has_qoutation = false;
```
Comment (balakrishna-binary) [inline src/Translation.js:38]: typo

File: src/Constant.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ export const drawTools = {
     vertical:   { id: 'vertical',    text: t.translate('Vertical [num]'), icon: DrawToolsVerticalIcon },
 };
 
-export const IndicatorsTree = [
+export const IndicatorsTree = () => [
```
Comment (balakrishna-binary) [inline src/Constant.js:59]: can we change this to camel case? `getIndicatorsTree`

File: src/Translation.js (hunk 1 above)
Comment (shayan-binary-2) [reply src/Translation.js]: @balakrishna-binary fixed

File: src/Constant.js (hunk 2 above)
Comment (shayan-binary-2) [reply src/Constant.js]: @balakrishna-binary thanks, just rename it

**Review by balakrishna-binary — APPROVED** [review]

**Review by zaki-binary — APPROVED** [review]

**Pull request merged**